buildSrc/src/main/java/org/springframework/boot/build/ConventionsPlugin.java

```diff
@@ -123,53 +116,81 @@ public void apply(Project project) {
 
 	private void applyJavaConventions(Project project) {
 		project.getPlugins().withType(JavaBasePlugin.class, (java) -> {
-			project.getPlugins().apply(TestFailuresPlugin.class);
+			applyTestFailuresPlugin(project);
```

Comment:
Given that applying the plugin is a single line and wrapping it in a method doesn't make the intent any clearer, I'd prefer to keep it inlined here.